src/XRP/wallet.ts

```diff
@@ -27,6 +27,23 @@ export interface WalletGenerationResult {
  * A wallet object that has an address and keypair.
  */
 class Wallet {
+  private readonly publicKey: string
+  private readonly privateKey: string
+  private readonly test: boolean
+
+  /**
+   * Create a new Wallet object.
+   *
+   * @param publicKey - The given public key for the wallet.
+   * @param privateKey - The given private key for the wallet.
+   * @param test - Whether the address is for use on a test network, defaults to `false`.
+   */
+  public constructor(publicKey: string, privateKey: string, test = false) {
```

Comment:
Moved the constructor because the [member-ordering](https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/member-ordering.md) rule defines an order that class members should appear in (and constructors come before any methods).